### ISSUE TITLE
plan for multi-platform integration

### ISSUE THREAD

**Issue by rahulbot** [mitmedialab/MediaCloud-Web-Tools#1367]
One major goal over the next 6 months is to integrate data from multiple web platforms, and tools for analyzing each, and analyzing them together.  We need to start planning for how we will acquire this data, store it, support searching, and support analyzing it.

The prioritized list I heard from the last all-hands meeting, and conversations since then, is:
* Twitter (via Crimson Hexagon)
* YouTube
* Reddit (via PushShift)
* Radio (via LSM podcast transcription)

Other platforms include Facebook (via CrowdTangle), 4chan, 8chan, WhatsApp, TV, Radio, Instagram.

The technical considerations for these move us from the model of media sources/stories to one that could potentially include platforms, authors, threaded conversations, audio, video, photos, and more. Not to mention the shift from gathering the open web to gathering from private platforms (that might not want us to gather their content).

This an overwhelming amount of potential work and research, so we need to prioritize it with the research folks at BK and ML.

**Comment by cindyloo** [comment]
Meeting with Aashka yesterday. We identified a few use cases and went through them.

Her viewpoint is that it is better to enable the user to iterate/update each individual platform seed/boolean queries before launching the creation of the topic. This is the other path than what @rahulbot and I discussed, which is creating the topic adding one platform at a time.

I'll create a sketch that offers each flow, and we can discuss as a team in the next couple of weeks
